tests/dummy/lib/outdated/app/utils/floor-outdated.js

```diff
@@ -1,5 +1 @@
-import flooring from 'npm:flooring';
-
-export default function() {
-  return flooring();
-}
+export { default } from 'outdated/utils/floor-type';
```

Comment:
This is _intentionally_ different than the addon module and should not re-export the module from the addon.